### PR TITLE
generate placeholders for math section

### PR DESCRIPTION
**Pull request by Objectivitix**
approve

### CODE REVIEW

**Comment by Kronifer** [comment]
dude this is a fork we arent managing by approvals

**Comment by Objectivitix** [comment]
bruh- decentralized system

File: bot/resources/evergreen/trivia_quiz.json [hunk 1]
```diff
@@ -269,5 +269,160 @@
...
+    },
+    {
+      "id": 209,
+      "question": 209,
+      "answer": 209
+    },
+    {
+      "id": 210,
+      "question": 210
+      "answer": 210
```
Comment (Shivansh-007) [inline bot/resources/evergreen/trivia_quiz.json:325]: Smh, bad reviewing, this has a bug
```suggestion
      "question": 210,
      "answer": 210
```

File: bot/resources/evergreen/trivia_quiz.json [hunk 2]
```diff
@@ -269,5 +269,160 @@
...
+    },
+    {
+      "id": 223,
+      "question": 223,
+      "answer": 223
+    },
+    {
+      "id": 224,
+      "question": 224
+      "answer": 224
```
Comment (Shivansh-007) [inline bot/resources/evergreen/trivia_quiz.json:395]:
```suggestion
      "question": 224,
      "answer": 224
```

**Pull request merged**